packages/docusaurus-theme-classic/src/theme/Navbar/index.js

```diff
@@ -43,10 +44,16 @@ function NavLink({
             isNavLink: true,
             activeClassName,
             to: toUrl,
-            ...(activeBasePath
+            ...(activeBasePath || activeBaseTest
               ? {
-                  isActive: (_match, location) =>
-                    location.pathname.startsWith(activeBaseUrl),
+                  isActive: (_match, location) => {
+                    if (activeBaseTest) {
+                      const activeBaseReg = new RegExp(activeBaseTest);
```

Comment:
If we rename the option to `activeBaseRegex`, we can completely refuse from separate const.

```js
if (activeBaseRegex) {
  return new RegExp(activeBaseRegex).test(location.pathname);
}
```

It is OK?

Or we might use ternary expression at all, like that:

```js
isActive: (_match, location) => activeBaseTest ?
  new RegExp(activeBaseRegex).test(location.pathname) : 
  location.pathname.startsWith(activeBaseUrl)
```

Reply:
That makes a lot more sense! I'll change it to the ternary.